doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -504,6 +504,13 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-9-0 v2-8-0 v2-7-0
+
+KRKNWK-19915: Total operational hours are saved to NVM with interval defined by :kconfig:option:`CONFIG_CHIP_OPERATIONAL_TIME_SAVE_INTERVAL` Kconfig option.
+  Certification test case TC-DGGEN-2.1 requires checking the value after 1 hour, restarting the DUT, and verifying if the value has not changed, causing failures if given option is higher than 1.
+
+  **Workaround:** Manually cherry-pick and apply the commit with the fix to ``sdk-connectedhomeip`` (commit hash: ``89e823e5d8065ce85a6e6b4c205e11db413c4535`` from the `upstream Matter SDK repository <https://github.com/project-chip/connectedhomeip>`_).
```

Comment:
I would like also to mention two possible workarounds. The second one is to uncheck the PICS and re-generate the zap files. Michał should be able to help here.

Reply:
added the second workaround